.goreleaser

```diff
@@ -0,0 +1,70 @@
+project_name: root-signing
+
+env:
+  - GO111MODULE=on
+  - CGO_ENABLED=0
+  - COSIGN_EXPERIMENTAL=true
+
+# Prevents parallel builds from stepping on eachothers toes downloading modules
+before:
+  hooks:
+    - go mod tidy
+    - /bin/bash -c 'if [ -n "$(git --no-pager diff --exit-code go.mod go.sum)" ]; then exit 1; fi'
+
+gomod:
+  proxy: true
+
+sboms:
+  - artifacts: binary
+
+builds:
+  - id: root-verify
+    binary: root-verify-linux-{{ .Arch }}
+    no_unique_dist_dir: true
+    main: ./cmd/verify
+    goos:
+      - linux
+    goarch:
```

Comment:
Should we just target the architecture needed for GHA for now? Doesn't hurt to have more though

Reply:
yeah! rather limit usage for now